### ISSUE TITLE
Is the custom ssd model used right?

### ISSUE THREAD

**Issue by wyattsuen** [dmlc/gluon-cv#908]
Hi, I use the model ssd_300_vgg16_atrous_custom to compare with my hand writing simple model. I think the standard model must be better than my simple model. But the predicted bounding box of standard model seems random and bad. I don't know what's wrong with my training, so I post my whole code here. All code is following the official tutorial.
``` 
train_dataset=LstDetection('./train.lst', root='e:/root')

width, height=300, 300
batch_size=20
num_workers=0

net=model_zoo.get_model('ssd_300_vgg16_atrous_custom', classes=['class1'], pretrained=False, pretrained_base=False)
net.initialize()

x=nd.zeros(shape=(1,3,height,width))
with autograd.train_mode():
    cls_preds, box_preds, anchors=net(x)

train_transform=presets.ssd.SSDDefaultTrainTransform(width, height, anchors)
batchify_fn=Tuple(Stack(), Stack(), Stack())
train_loader=gdata.DataLoader(
    train_dataset.transform(train_transform), 
    batch_size=batch_size, 
    shuffle=True, 
    batchify_fn=batchify_fn, 
    last_batch='rollover', 
    num_workers=num_workers
)
#I checked and plot the batch in train_loader, right. So above code is checked.
mbox_loss=SSDMultiBoxLoss()
trainer=gluon.Trainer(net.collect_params(), 'adam', {'learning_rate':0.001}) #I test learning_rate 0.01, 0.001, 0.0001 and more

for epoch in range(100):
    for i, batch in enumerate(train_loader):
        with autograd.record():
            cls_pred, box_pred, anchors=net(batch[0])
            sum_loss, cls_loss, box_loss=mbox_loss(cls_pred, box_pred, batch[1], batch[2])
        autograd.backward(sum_loss)
        trainer.step(1)
        l=sum_loss[0].mean().asscalar()
        print('epoch {0}, batch {1}, sum_loss mean {2}'.format(epoch, i, l))
```
The loss goes to near 0.18 to 0.2 and won't converge. Then I  use following code to predict the bounding boxes.
```
class_IDs, scores, bounding_boxes = net(x)
```
The result returns top 100. All class_IDs are 0(no -1) and all scores are below 0.5. That's not normal. And when I plot the bounding_boxes, all boxes are random distributed in the image.
I'm very sure my images dataset can train my simple ssd model good. So can anyone point out why the standard model like ssd_300_vgg16_atrous_custom fail?

**Comment by zhreshold** [comment]
You are using pretrained_base=False which means it's not even pretrained on imagenet, which means if your training dataset is not big enough, the model won't converge.
What I would do is 
```
net=model_zoo.get_model('ssd_300_vgg16_atrous_custom', classes=['class1'], transfer='coco')
```

**Comment by wyattsuen** [comment]
@zhreshold I used to think the more complex the model the easier it converge for simple dataset. Now you correct my idea. Thanks very much.
